source/stdlib-reorg.rst

```diff
@@ -23,12 +23,11 @@ you should import ``six.moves``::
 
     from six.moves.configparser import ConfigParser
 
-A list of all renamed modules is included in `six documentation`_.
+A list of all renamed modules is included in
+`six documentation <https://pythonhosted.org/six/#module-six.moves>`_.
```

Comment:
The actual error here is that "six documentation" is used two times (and one of them is wrong).